apps/web/src/views/BuyCrypto/constants.ts

```diff
@@ -149,12 +152,19 @@ export const chainIdToTransakNetworkId: { [id: number]: string } = {
   [OnRampChainId.BTC]: 'mainnet',
 }
 
+export const chainIdToTopperNetworkId: { [id: number]: string } = {
+  [OnRampChainId.ETHEREUM]: 'ethereum',
+  [OnRampChainId.ARBITRUM_ONE]: 'arbitrum',
+  0: 'bitcoin',
```

Comment:
why bitcoin is 0?

Reply:
it does have a chanid so its a work around so its works with out token list chaind mapings system